react/features/polls/components/AbstractPollAnswer.tsx

```diff
@@ -92,7 +92,8 @@ const AbstractPollAnswer = (Component: ComponentType<AbstractProps>) => (props:
             type: COMMAND_NEW_POLL,
             pollId,
             question,
-            answers
+            answers: answers.map(answer => answer.name),
+            poll
```

Comment:
don't think we need to send the `poll` object - it will be ignored as it's not recognised on prosody

Reply:
Right. Forgot to delete this one.